pkg/sdk/go/bootstrap.go

```diff
@@ -216,6 +231,9 @@ func (sdk mgSDK) UpdateBootstrapConnection(id string, channels []string, token s
 }
 
 func (sdk mgSDK) RemoveBootstrap(id, token string) errors.SDKError {
+	if id == "" {
```

Comment:
Also check for `Bootstrap` and `BootstrapSecure` that the `externalID` is not empty